service/controller/v5/resource/blobobject/create.go

```diff
@@ -0,0 +1,97 @@
+package blobobject
+
+import (
+	"context"
+	"fmt"
+	"github.com/giantswarm/azure-operator/service/controller/v5/blobclient"
+	"github.com/giantswarm/azure-operator/service/controller/v5/key"
+	"github.com/giantswarm/microerror"
+)
+
+func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange interface{}) error {
+	customObject, err := key.ToCustomObject(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	containerObjectToCreate, err := toContainerObjectState(createChange)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	storageAccountsClient, err := r.getAccountsClient()
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	var blobClient *blobclient.BlobClient
+	{
+		c := blobclient.Config{
+			ContainerName:         key.BlobContainerName(),
+			GroupName:             key.ClusterID(customObject),
+			StorageAccountName:    key.StorageAccountName(customObject),
+			StorageAccountsClient: storageAccountsClient,
+		}
+
+		blobClient, err = blobclient.New(c)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+	}
+
+	err = blobClient.Boot(ctx)
```

Comment:
At this point it is not really clear the client is needed at all. In general, let's make sure the work we do in resources is always useful. In case the managed storage thingy is already created, we boot the client and what not even if this is not necessary. Maybe will be addressed anyway when the other concern above is addressed, but just wanted to emphazise that again.